lib/ansible/modules/cloud/ovirt/ovirt_cluster.py

```diff
@@ -485,7 +485,9 @@ def build_entity(self):
                 name=self.param('network'),
             ) if self.param('network') else None,
             cpu=otypes.Cpu(
-                architecture=self.param('cpu_arch'),
+                architecture=otypes.Architecture(
+                    self.param('cpu_arch')
+                ),
```

Comment:
Actually, can you change it to:

```python
achitecture=otypes.Architecture(
  self.param('cpu_arch')
) if self.param('cpu_arch') else None,
```

So if `cpu_arch` parameter isn't passed we don't try to sent empty `architecture` parameter.

Reply:
Yeah, i guess that case  would be handled by this fix, which is in place already. 

https://github.com/ansible/ansible/pull/34446

Reply:
Actually, that's different case. The code above I've sent:

```python
achitecture=otypes.Architecture(
  self.param('cpu_arch')
) if self.param('cpu_arch') else None,
```

Ensures it sent `<achitecture/>` xml element when creating/updating cluster only when user passes `cpu_arch` parameter to Ansible. The current code you've sent will sent `<achitecture/>` element also when user don't `cpu_arch` parameter.

Reply:
Cool got it, Thanks! changed it.